eZ/Publish/API/Repository/URLWildcardService.php

```diff
@@ -84,6 +86,13 @@ public function load(int $id): UrlWildcard;
      */
     public function loadAll(int $offset = 0, int $limit = -1): iterable;
 
+    /**
+     * Find URLWildcards.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
```

Comment:
Unreported exception:
```suggestion
     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
     */
```